lab_1/main.py

```diff
@@ -49,7 +77,26 @@ def get_top_n_words(freq_dict: dict, top_n: int) -> list:
     top_n = 1
     --> ['happy']
     """
-    pass
+    if not isinstance(freq_dict, dict) or not isinstance(top_n, int):
+        return []
+    else:
```

Comment:
Shorten the function by one line. Please, explain how and why it can be shortened.